doc/manual/src/command-ref/nix-store.md

```diff
@@ -62,28 +62,28 @@ have an effect.
 
 ## Description
 
-The operation `--realise` essentially “builds” the specified store
-paths. Realisation is a somewhat overloaded term:
-
-  - If the store path is a *derivation*, realisation ensures that the
-    output paths of the derivation are [valid](../glossary.md) (i.e.,
-    the output path and its closure exist in the file system). This
-    can be done in several ways. First, it is possible that the
-    outputs are already valid, in which case we are done
-    immediately. Otherwise, there may be [substitutes](../glossary.md)
-    that produce the outputs (e.g., by downloading them). Finally, the
-    outputs can be produced by running the build task described
-    by the derivation.
-
-  - If the store path is not a derivation, realisation ensures that the
-    specified path is valid (i.e., it and its closure exist in the file
-    system). If the path is already valid, we are done immediately.
-    Otherwise, the path and any missing paths in its closure may be
-    produced through substitutes. If there are no (successful)
-    substitutes, realisation fails.
-
-The output path of each derivation is printed on standard output. (For
-non-derivations argument, the argument itself is printed.)
+Ensure that the given [store paths] are [valid].
+
+Realisation of a store path works as follows:
+- If the path is already valid, do nothing.
+- If the path leads to a [store derivation], realise its [output paths]:
+ - Try to fetch the associated [store objects] in the paths' [closure] from a [substituter].
+ - Produce the required store objects by first realising all dependencies and then running the derivations' build instructions.
```

Comment:
```suggestion
 - Produce the required store objects by first realising the derivation's dependencies' outputs and then running the derivation's build instructions.
```

This is yet another form of realisation, where we only substitute the desired output, but may need to build the whole thing if the output wasn't substitutable.
We've fixed a bug in that area in the last release.

`the derivation's build instructions.`

singular; not the dependencies.